src/applications/ivc-champva/10-7959f-1/tests/e2e/10-7959f-1-FMP.cypress.spec.js

```diff
@@ -7,14 +7,20 @@ import formConfig from '../../config/form';
 import manifest from '../../manifest.json';
 import {
   fillAddressWebComponentPattern,
-  fillDateWebComponentPattern,
   fillTextWebComponent,
+  reviewAndSubmitPageFlow,
+  verifyAllDataWasSubmitted,
+  getAllPages,
 } from '../../../shared/tests/helpers';
 
+import mockFeatureToggles from './fixtures/mocks/featureToggles.json';
+
+const ALL_PAGES = getAllPages(formConfig);
```

Comment:
Rather than use hardcoded strings for the various page paths, access the keys on this object. That way, if the actual path strings are updated, we'll still be getting the correct pages in the test.